src/main/java/Core/Server.java

```diff
@@ -0,0 +1,46 @@
+package Core;
+
+import Core.Message;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Server {
+
+    private PrintStream stdOut;
+    private ServerSocket serverSocket;
+    private List<Socket> clients;
```

Comment:
Domain question: Is a `List<Socket>` a domain concept? The fact that you called the variable `clients` and not `connectedSockets` or  even`sockets` hints at the fact that you _think_ of this list as _the clients_ but your type system/domain does not expose it as such.
Why not do something like `List<Client>`? Or even better, `Clients`!
Then you coudld have `Clients connectedClients = ...` and then `connectedClients.broadcast(....)`!
Elevate concepts to classes and functions/methods on those classes. Use your domain! 😄